vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -128,12 +131,16 @@ IAsyncAction WinRTWebSocketResource::PerformConnect() noexcept {
     } else {
       if (self->m_errorHandler) {
         auto error = winrt::hresult_error(result, winrt::hresult_error::from_abi);
-        self->m_errorHandler({winrt::to_string(error.message()), ErrorType::Connection});
+        std::stringstream hexCode;
+        hexCode << "[0x" << std::hex << error.code() << "] ";
+        self->m_errorHandler({hexCode.str() + winrt::to_string(error.message()), ErrorType::Connection});
```

Comment:
instead of tying the error string with the error code, maybe the Error struct should keep the error code number in it (and can be inspected independently of the error message string)

Reply:
That would make the interface assume it has both a numeric error and a message string.
While I admit it's VERY unlikely to switch away from the C++/WinRT implementation, it's cleaner to keep that detail separate.

This also keeps the change uninvasive.

Reply:
there's always a numeric error though, isn't there?

Reply:
It’s up to the framework.

Moreover, the JS layer only expects a single string, and the native module doesn’t have a use for a numeric code.

There is no use case that justifies changing the interface at this point.